src/v2i-hub/ImmediateForwardPlugin/README.md

```diff
@@ -70,7 +70,39 @@ This plugin has several configuration parameters. Below these are listed out as
                     { "tmxType": "SSM-P", "sendType": "SSM", "psid": "0x8002", "channel": 183 },
                     { "tmxType": "SDSM", "sendType": "SDSM", "psid": "0x8010", "channel": 183 }
                 ]
+            },
+            
+        {
+            "name": "Yunnex",
+            "rsuSpec": "NTCIP1218",
+            "address": "192.168.55.20",
```

Comment:
Is this real ip address? Can we use a fake one?

Reply:
Why

Reply:
Sure